src/main/java/seedu/storemando/logic/commands/ReminderCommand.java

```diff
@@ -19,23 +19,49 @@
         + "Parameters: Days/Weeks (must be an integer) [TimeUnitKeyWord] (days/weeks (day/week accepted for -1/0/1)\n"
         + "Example: \n- " + COMMAND_WORD + " 3 days\n- " + COMMAND_WORD + " 1 week";
 
-    public static final String MESSAGE_SUCCESS = "Display all expiring items";
+    public static final String MESSAGE_SUCCESS_EXPIRING_ITEM = "Display all items that are expiring in %d %s or has "
```

Comment:
for this I feel you should use within instead of in

Reply:
All right noted!